### ISSUE TITLE
Call out need for additional build config

### ISSUE THREAD

**Issue by drouillard** [fizzed/ninja-rocker#2]
Similar to #1  (which I did not see until I had fixed the issue below)

I was trying to set up a "hello world" type app using a simple Ninja app I had created following Ninja's framework tutorials. I was able to get rocker templates to generate without too much of an issue but spent many hours with errors such as

```
[qtp1018937824-17] ERROR c.f.n.rocker.TemplateEngineRocker - Unable to handle renderable not of type: class views.ApplicationController.index
```

The fix was easy enough - adding the following to the build plugin

```
 <configuration>
    <extendsClass>com.fizzed.ninja.rocker.NinjaRockerTemplate</extendsClass>
 </configuration>
```

With final result looking like

```
<plugin>
<groupId>com.fizzed</groupId>
<artifactId>rocker-maven-plugin</artifactId>
<version>0.9.0</version>
<executions>
    <execution>
        <id>generate-rocker-templates</id>
        <phase>generate-sources</phase>
        <goals>
            <goal>generate</goal>
        </goals>
        <configuration>
            <extendsClass>com.fizzed.ninja.rocker.NinjaRockerTemplate</extendsClass>
        </configuration>
    </execution>
  </executions>
 </plugin>
```

A few things I would suggest
1. Update ninja-rocker READMe with explicit call out to this. The instructions say to use Rockers build plug-in XML, but that is not sufficient
2. Improve error message if possible. Unable to handle renderer did not make me think this was the fix.
3. A simpler hello world app outside of the demo. 
   1. The simplest demo page still uses templates that call templates
   2. Demo's POM having a parent and project being nested a bit confusing

I had spent a lot of my time thinking I had template specific issues. Such as the case in #1 thinking N needed to be passed through. I will take a shot at the third item - creating a demo app. I like the existing demo app and don't think it needs to change. Just a simpler app if people are having even the most basic of basic issues, such as getting 'Hello World' to display.

**Comment by drouillard** [comment]
Started porting over the default Ninja Framework Project and including a really simple hello world page

https://github.com/drouillard/simple-ninja-rocker

Hello world page is done, need to work on porting over the existing ftl templates

**Comment by jjlauer** [comment]
Addressed in v0.9.1 module.  Added more error checking with proposed solutions in exception messages.
